src/lib/components/Card/Card.svelte

```diff
@@ -1,58 +1,38 @@
 <script lang="ts">
   import "./Card.scss";
-  import Button from "$lib/components/Button";
-  import Image from "$lib/components/Image";
   import classNames from "$lib/util/classNames";
-  import type { Variant } from "./cardOptions";
-
-  export let variant: Variant = "primary";
-
-  const variantClassesDict: Record<Variant, string> = {
-    primary: $$slots.image ? "usa-card--flag" : "",
-    secondary: "",
-  };
-  $: variantClass = variantClassesDict[variant];
 
   let className = "";
   export { className as class };
-  $: classes = classNames("usa-card", variantClass, className);
+  $: classes = classNames("usa-card", $$slots.image ? "usa-card--flag" : null, className);
 </script>
 
-<li class={`${classes}`}>
+<li class={classes}>
   <div class="usa-card__container">
     {#if $$slots.header}
       <div class="usa-card__header">
-        <slot name="header">
-          <h2 class="usa-card__heading">Default header</h2>
-        </slot>
+        <!-- Use the appropriate section heading element, likely <h2> or <h3>, based on page structure -->
```

Comment:
Appreciate these comments you added!

Reply:
Yeah, since there's not a way to directly provide descriptions of a named slot in storybook, and since to my knowledge, you could conceivably put any element or component within a slot, this was the best method I landed on to make things slightly self-documenting